modules/core/src/main/forum.scala

```diff
@@ -3,16 +3,38 @@ package forum
 
 import lila.core.id.{ ForumCategId, ForumPostId, ForumTopicId, TeamId }
 import lila.core.userId.*
+import lila.core.bus
 
 import reactivemongo.api.bson.Macros.Annotations.Key
 
-case class CreatePost(post: ForumPostMini)
+sealed class BusForumPost
+object BusForumPost:
+  given busForumPostChannel: bus.WithChannel[BusForumPost] with
```

Comment:
We can use anonymous given here:
```suggestion
  given bus.WithChannel[BusForumPost] with
```